Content/posts/embracing-the-ecosystem.md

```diff
@@ -0,0 +1,23 @@
+---
+date: 2024-12-08 14:00
+description: Embracing more parts of the Swift on Server ecosystem as we progress towards Vapor 5
+tags: framework, vapor5, ecosystem
+image: /static/images/posts/Vapor5.png
+author: Tim
+authorImageURL: /author-images/tim.jpg
+---
+# Embracing the Ecosystem
+
+As the march to Vapor 5 continues, we are starting to see what our packages will look like in the modern ecosystem. Part of this involves integrating the many packages that are becoming integral to Swift on the server.
+
+[JWTKit](https://github.com/vapor/jwt-kit) is the prime example of this, with the new major version taking advantage of Swift 6, [Swift Testing](https://github.com/swiftlang/swift-testing), [Benchmark](https://github.com/ordo-one/package-benchmark), and the latest improvements in [Swift Crypto](https://github.com/apple/swift-crypto). JWTKit v5 was an important release that removed a vended copy of BoringSSL to improve build times for everyone building it and was built with Swift Concurrency in mind from the start.
```

Comment:
"Benchmark" sounds a bit weird like that but not sure what's better ... possibly "Package-Benchmark" but that still does sound a bit weird.